modules/aggregation-layers/src/screen-grid-layer/screen-grid-layer-vertex.glsl.js

```diff
@@ -67,8 +61,9 @@ vec4 quantizeScale(vec2 domain, vec4 range[RANGE_COUNT], float value) {
 void main(void) {
   vSampleCount = instanceCounts.a;
 
-  float weight = instanceCounts.r ;
-  float maxWeight = aggregationData.maxCount.r;
+  float weight = instanceCounts.r;
+  float maxWeight = texture2D(maxTexture, vec2(0.5)).r;
```

Comment:
Should we just send the max value as a uniform? My experience is that texture sampling is must slower.

Reply:
When using GPU Aggregation, we would need to perform a readPixel which will cause a CPU and GPU sync, and this will impact performance negatively for all GPU heavy use cases.